znai-docs/documentation/flow/shortcuts.md

```diff
@@ -0,0 +1,3 @@
+# Keyboard shortcuts
+
+:include-table: shortcuts.csv {columns: ["Action", "Shortcut"]}
```

Comment:
not sure but I think you can skip the columns definitions

Reply:
Removed.